ntc_templates/templates/alcatel_sros_show_service_id_interface.textfsm

```diff
@@ -0,0 +1,32 @@
+Value Required INTERFACE_NAME (\S+)
+Value Required ADMIN_STATE (Up|Down)
+Value Required OPER_STATE_V4 (Up|Down)
+Value Required OPER_STATE_V6 (Up|Down)
+Value Required TYPE (\S+)
+Value Required PORT_SAP_ID (\S+)
+Value List IP_ADDRESS (\S+)
+Value List PFX_STATE (\S+)
+
+Start
+  ^=+
+  ^Interface\s+Table
+  ^Interface-Name\s+Adm\s+Opr\(v4\/v6\)\s+Type\s+Port/SapId\s*$$ -> Interface
+  ^\s*$$
+  # added to avoid errors with trucated values alert message
+  ^\S+
+  ^. -> Error
+
+Interface
+  ^\s+IP-Address\s+PfxState
+  ^-+
+  ^.*?\s+(Up|Down) -> Continue.Record
+  ^${INTERFACE_NAME}\s+${ADMIN_STATE}\s+${OPER_STATE_V4}\/${OPER_STATE_V6}\s+${TYPE}\s+${PORT_SAP_ID} 
+  ^\s+${IP_ADDRESS}\s+${PFX_STATE}
+  ^Interfaces -> Done
```

Comment:
```suggestion
  ^Interfaces\s*:\s+\d+\s*$$
```

We are at the end of the data, so I don't see a need to move to another state to move to the implicit `End -> Record` state.